pyproject.toml

```diff
@@ -119,26 +119,9 @@ docs = ["mkdocs", "mkdocs-material", "mkdocs-drawio-file >= 1.5.2"]
 ato = "atopile.cli.cli:main"
 faebryk = "faebryk.tools.main:__main__"
 
-[tool.hatch]
```

Comment:
nit: I think we've maintained a chapter-like styling through this PR, which this conforms with